lib/liquid/combinators/tags/custom_block.ex

```diff
@@ -36,6 +26,10 @@ defmodule Liquid.Combinators.Tags.CustomBlock do
     |> tag(:custom_block)
   end
 
+  @doc """
+  Check if the closed tag name correspond to the opened tag name.
```

Comment:
Checks instead of Check. Review all documentation, this error is repeatead

Reply:
done